doc/cpp/static-grpc-object.md

```diff
@@ -0,0 +1,38 @@
+# gRPC Objects with static storage duration
+
+In general, it's discouraged to define gRPC objects with static storage
```

Comment:
How about instead of "discouraged", we say "not supported"?

Reply:
SGTM.